tsconfig.json

```diff
@@ -3,5 +3,6 @@
     "include": ["./src/**/*.ts", "./src/**/*.tsx"],
     "compilerOptions": {
         "lib": ["DOM", "ES2020"]
-    }
+    },
+    "exclude": ["node_modules", "**/*test.ts"]
```

Comment:
I don't think we want to exclude test files, otherwise there's not much point in having them written in Typescript.